skaffold.yaml

```diff
@@ -28,6 +28,7 @@ deploy:
         chartPath: charts/substra-backend
         valuesFiles:
           - examples/values/backend-org-1.yaml
+          - charts/substra-backend/resources.yaml
```

Comment:
I am not sure I am following. Is it not prod values that are in the `resources.yaml` file?

Reply:
The values are the ones that we currently apply by default. It is up to us to decide if we put prod values (so with highest requests) or if we want to keep these low requests. Also, il was to test that it was working as expected. We can also completely remove it from skaffold but I don't know how to ensure that this file stay up to date with CI